test/crowdsale/TimedCrowdsale.test.js

```diff
@@ -73,5 +73,67 @@ contract('TimedCrowdsale', function ([_, investor, wallet, purchaser]) {
         await shouldFail.reverting(this.crowdsale.buyTokens(investor, { value: value, from: purchaser }));
       });
     });
+
+    describe('extending closing time', function () {
+      it('should not reduce duration', async function () {
+        (await this.crowdsale.isOpen()).should.equal(false);
+        await shouldFail.reverting(this.crowdsale.send(value));
```

Comment:
I take it these are a sort of double-check that the crowdsale is indeed closed? I'd get rid of them and just trust `isOpen` - the place to do these sort of tests would be `Crowdsale.test.js`, not here, and they decrease readability. If we can't trust submodules to behave properly, the suite will become full of redundant tests.